scripts/src/models/orders.ts

```diff
@@ -119,6 +119,20 @@ export class Order extends CreationDateModel {
 		return query.getOne() as Promise<T | undefined>;
 	}
 
+	public static async updateUncompletedOrders<T extends Order>(): Promise<void> {
+		const now = moment().format("YYYY-MM-DD HH:mm:ss.SSS");
+		const errorMessage = TransactionTimeout().toString();
```

Comment:
If you're looking for the message only then you can do:
```
const errorMessage = TransactionTimeout().message;
```
And then you don't need to `JSON.stringify`.  If you keep it this way, then there's no need to first use `toString()` and then `JSON.stringify`, one should be enough.

Reply:
The stringify is redundent here. thanks.